keentools/geotracker/camera_input.py

```diff
@@ -114,13 +116,17 @@ def first_frame(self) -> int:
         geotracker = get_current_geotracker_item()
         if not geotracker:
             return 1
-        return geotracker.precalc_start
+        if not geotracker.precalcless:
+            return geotracker.precalc_start
```

Comment:
Remove this. If precalc is used, precalc range is taken from file, not from image input

Reply:
ok